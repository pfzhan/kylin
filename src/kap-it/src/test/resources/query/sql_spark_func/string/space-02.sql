select '>' + space(LSTG_SITE_ID)+ '<' from (
select LSTG_SITE_ID from TEST_KYLIN_FACT where LSTG_SITE_ID=15 limit 1)