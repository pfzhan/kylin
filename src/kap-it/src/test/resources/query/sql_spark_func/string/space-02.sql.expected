select '>               <' from TEST_KYLIN_FACT limit 1